data/mods/innawood/recipes/other_parts.json

```diff
@@ -30,5 +30,28 @@
       { "proficiency": "prof_plumbing", "required": false, "time_multiplier": 3, "fail_multiplier": 1.5 }
     ],
     "components": [ [ [ "pipe", 1 ] ], [ [ "scrap", 1 ] ], [ [ "plastic_chunk", 1 ] ], [ [ "pipe_fittings", 1 ] ] ]
+ },
+ {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "well_pump",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_OTHER",
+    "skill_used": "fabrication",
+    "difficulty": 5,
+    "time": "6 h 30 m",
+    "autolearn": true,
+    "book_learn": [ [ "textbook_fabrication", 4 ], [ "manual_fabrication", 4 ] ],
+    "using": [ [ "blacksmithing_standard", 8 ] ],
+    "proficiencies": [
+      { "proficiency": "prof_metalworking" },
+      { "proficiency": "prof_blacksmithing" },
+      { "proficiency": "prof_toolsmithing" }
+    ],
+    "tools": [ [ [ "swage", -1 ] ] ],
+    "components": [
+      [ [ "steel_chunk", 6 ], [ "scrap", 18 ] ], [ [ "pipe", 2 ] ],
```

Comment:
Looks like you need a newline to change this line to to 

      [ [ "steel_chunk", 6 ], [ "scrap", 18 ] ],
      [ [ "pipe", 2 ] ],

For future reference, see [here](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_STYLE.md) for json style guide. Specifically you might want to use [this](http://dev.narc.ro/cataclysm/format.html) tool to automatically format your json. Just paste the entire file content in there, click lint, and put it back in your file.

Once you have made your changes, you should be able to make a new commit with them and then push them up. This pull request would then be automatically updated.

Reply:
Thanks!